src/templates/dotnet/executeDotnetTemplateCommand.ts

```diff
@@ -61,7 +61,7 @@ async function getFramework(context: IActionContext, workingDirectory: string |
         }
 
         // Prioritize "LTS", then "Current", then "Preview"
-        const netVersions: string[] = ['6.0', '5.0', '3.1'];
+        const netVersions: string[] = ['7.0', '6.0', '5.0', '3.1'];
```

Comment:
.NET Core 3.1 is EOL today. We may as well remove it. Also, 6.0 is LTS, so based on the comment shouldn't it be 6.0, 7.0, 5.0?

Reply:
Ahh yes I will make that change and remove 3.1 😊